src/components/TaskList.svelte

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+  import type { TodoItem } from '../types/item';
+  import * as animateScroll from 'svelte-scrollto';
+  import { onMount } from 'svelte';
+  import ListItem from './ListItem.svelte';
+
+  export let taskListName: string;
+  export let items: TodoItem[];
+  export let highlightItemIndex = -1;
+  export let showOnlyItemIndex = -1;
+  export let scrollingInterval = 5000;
+  export let scrollingDuration = 2000;
+  export let layout = 'full';
+
+  // TODO Either add scrolling up / down
+  // TODO or reverse items list
+
+  let itemsListElement;
+
+  onMount(() => {
+    var scrollToTop = false;
+
+    setInterval(() => {
+      if (scrollToTop) {
+        animateScroll.scrollToTop({
+          container: itemsListElement,
+          duration: scrollingDuration
+        });
+      } else {
+        animateScroll.scrollToBottom({
+          container: itemsListElement,
+          duration: scrollingDuration
+        });
+      }
+
+      scrollToTop = !scrollToTop;
+    }, scrollingInterval);
+  });
+</script>
+
+<style>
+  .nes-container {
+    height: calc(100vh - 8px);
+  }
+
+  .nes-container.auto-layout {
+    height: 100%;
+    max-height: 100vh;
+  }
+
+  .items-holder {
+    height: 100%;
+    overflow-y: hidden;
+    padding-top: 0.5rem;
+    margin-left: -1rem;
+  }
+
+  .nes-container.with-title > .title {
+    margin-bottom: 0;
+  }
+</style>
+
+<div
+  class="nes-container {layout === 'auto' ? 'auto-layout' : ''} is-dark with-title">
```

Comment:
what do you think, maybe that would be a good way to also use the `class:name` shorthand of svelte, this could remove some of the inline conditions

Reply:
I've made changes to the ListItem and TaskList to use the svelte class directive shorthand.